documentation/2.0/content/userguide/target_env.md

```diff
@@ -179,6 +180,18 @@ This field specifies how the domain's `bin` directory contents should be handled
 
 This field specifies a name for use with the WDT_MODEL_SECRETS_NAME_DIR_PAIRS environment variable to identify administration credential Secrets for the domain. This is useful when those Secrets are stored in a directory that does not follow the `<directory>/<name>/<key>` convention. For more information about using the WDT_MODEL_SECRETS_NAME_DIR_PAIRS environment variable, see [Model tokens]({{< relref "/concepts/model#model-tokens" >}}).
 
+#### `use_persistent_volume`
+
+This field specifies if the domain is to be created for the Domain in PV [domain home source type](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/choosing-a-model/). If set to `true`, volume information will be added to the Kubernetes resource file that is generated.
+
+#### `additional_secrets`
+
+This field specifies a comma-separated list of secret types that are to be included in the Kubernetes resource file and the create secrets script. There is one secret type available:
```

Comment:
Is it "...list of secret types" or should it be "..list of secrets"?

Is runtime-encryption-secret a type or a secret?

Reply:
It's a type (key) - for the only existing case, we match the key and include a specific attribute / value in the resource file, and add a secret name that's not an exact match. Users can't put a random secret in here and expect to see it in the resource file or create secrets script.